lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -44,6 +46,8 @@ def activate(global_state, outgoing_queue)
           @rails_runner_client = addon.rails_runner_client
           outgoing_queue << Notification.window_log_message("Activating Tapioca add-on v#{version}")
           @rails_runner_client.register_server_addon(File.expand_path("server_addon.rb", __dir__))
+
+          handle_gemfile_changes
```

Comment:
I think something more descriptive is better here. Example:
```suggestion
        generate_gem_rbis if gemfile_changed
```

In this example `gemfile_changed` will contain `handle_gemfile_changes and some of `process_gemfile_changes`. I think something like this reads better because if I'm skimming through I know its intent.